crates/tabby-scheduler/src/lib.rs

```diff
@@ -20,6 +20,14 @@ use tracing::{debug, info};
 
 use crate::doc::SourceDocument;
 
+pub fn format_web_source(url: &str) -> String {
```

Comment:
this should goes to `tabby-webserver` and use `web_craw.id` for provider.

Reply:
I suppose we could also pass it in as an additional argument

Reply:
right - i suggest pass this as argument